packages/core/objects/_button-group.scss

```diff
@@ -0,0 +1,78 @@
+@use "../settings" as *;
+@use "../tools" as *;
+
+@include nhsuk-exports("nhsuk/core/objects/button-group") {
+  // Button groups can be used to group buttons and links together as a group.
+  //
+  // Within a button group:
+  //
+  // - links are styled to line up visually with the buttons, including being
+  //   centre-aligned on mobile
+  // - spacing between the buttons and links is handled automatically, including
+  //   when they wrap across multiple lines
+  .nhsuk-button-group {
+    $horizontal-gap: nhsuk-spacing(3);
+    $vertical-gap: nhsuk-spacing(3);
+
+    // These need to be kept in sync with the button component's styles
+    $button-border-radius: $nhsuk-border-width-form-element * 2;
+    $button-shadow-size: $button-border-radius;
+
+    $link-spacing: nhsuk-spacing(1);
+
+    // Flexbox is used to center-align links on mobile, align everything along
+    // the baseline on tablet and above, and to removes extra whitespace that
+    // we'd get between the buttons and links because they're inline-blocks.
+    //
+    // Ideally we'd use `gap` with flexbox rather than having to do it all with
+    // margins, but unfortunately the support isn't there (yet) and @supports
+    // doesn't play nicely with it
+    // (https://github.com/w3c/csswg-drafts/issues/3559)
```

Comment:
We're using `gap` for the new Header account section, so maybe support is ok for it now? https://github.com/nhsuk/nhsuk-frontend/pull/1058/files#diff-9872b34d22a8af3575af4913b03966d1994a4c9e966fddbaa977458158892ca1R119 cc @paulrobertlloyd

Reply:
We use `gap` in the header, but not as much as we once did; the navigation used it, but it meant items were bunched up on older versions of Safari locked-to-the-last-support-version-of-iOS. Probably something we’d want to test across a number of different devices.